### PR TITLE
Update serviceConnections

### PR DESCRIPTION
**Pull request by endakelly**
### Change description ###
Use subscription specific service connections so the correct state backend is used
Permissions have been updated as necessary
Update ptlsbox-00 version to match version currently running on the cluster

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: environments/aks/ptlsbox.tfvars [hunk 1]
```diff
@@ -1,6 +1,6 @@
 clusters = {
   "00" = {
-    kubernetes_cluster_version = "1.26"
+    kubernetes_cluster_version = "1.26.3"
```
Comment (thomast1906) [inline environments/aks/ptlsbox.tfvars:3]: Is this needed? I thought 1.26 - will determine latest 1.26.X

Comment (endakelly) [reply environments/aks/ptlsbox.tfvars:3]: Doesn't seem to. [Here](https://dev.azure.com/hmcts/PlatformOperations/_build/results?buildId=393388&view=logs&j=9529ead6-39ae-5319-1740-9d2035106bd6&t=89b2e481-5d95-5be1-d64b-e101e3d88784&l=358) shows it trying to perform a downgrade

Comment (thomast1906) [reply environments/aks/ptlsbox.tfvars:3]: @adusumillipraveen FYI^?

Comment (endakelly) [reply environments/aks/ptlsbox.tfvars:3]: Curiously, sbox does not exhibit the same behaviour despite also being on 1.26.3. None of the other environments are complaining about 1.25. Perhaps it's a random bug that can be ignored

Comment (endakelly) [reply environments/aks/ptlsbox.tfvars]: Reverted the patch version back to `1.26` and it's still proposing the same updates on ptlsbox.

**Review by timja — APPROVED** [review]

**Pull request merged**